<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Hello" name="com.testsuite.FooterTests.FooterTests" tests="18" failures="1" timestamp="20 Apr 2015 10:06:28 GMT" time="740.545" errors="1">
  <testcase name="CalaisIconCheck" time="40.026" classname="com.testsuite.FooterTests.FooterTests"/>
  <testcase name="Footertherankerblog" time="48.661" classname="com.testsuite.FooterTests.FooterTests"/>
  <testcase name="FooterPrivacy" time="28.140" classname="com.testsuite.FooterTests.FooterTests"/>
  <testcase name="Footertoprankers" time="123.843" classname="com.testsuite.FooterTests.FooterTests">
    <failure type="org.openqa.selenium.TimeoutException" message="Timed out waiting for page to load. (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 120.11 seconds
Build info: version: &#039;2.45.0&#039;, revision: &#039;32a636c&#039;, time: &#039;2015-03-05 22:01:35&#039;
System info: host: &#039;Hello&#039;, ip: &#039;192.168.1.2&#039;, os.name: &#039;Windows 7&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;6.1&#039;, java.version: &#039;1.7.0_10&#039;
Session ID: 32a2b781-c0d8-452a-9a8a-25a002ac81cd
Driver info: org.openqa.selenium.ie.InternetExplorerDriver
Capabilities [{platform=WINDOWS, javascriptEnabled=true, elementScrollBehavior=0, ignoreZoomSetting=false, enablePersistentHover=false, ie.ensureCleanSession=false, browserName=internet explorer, enableElementCacheCleanup=true, unexpectedAlertBehaviour=dismiss, version=11, ie.usePerProcessProxy=false, ignoreProtectedModeSettings=true, cssSelectorsEnabled=true, requireWindowFocus=false, initialBrowserUrl=http://localhost:17127/, handlesAlerts=true, ie.forceCreateProcessApi=false, nativeEvents=true, browserAttachTimeout=0, ie.browserCommandLineSwitches=, takesScreenshot=true}]">
      <![CDATA[org.openqa.selenium.TimeoutException: Timed out waiting for page to load. (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 120.11 seconds
Build info: version: '2.45.0', revision: '32a636c', time: '2015-03-05 22:01:35'
System info: host: 'Hello', ip: '192.168.1.2', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.7.0_10'
Session ID: 32a2b781-c0d8-452a-9a8a-25a002ac81cd
Driver info: org.openqa.selenium.ie.InternetExplorerDriver
Capabilities [{platform=WINDOWS, javascriptEnabled=true, elementScrollBehavior=0, ignoreZoomSetting=false, enablePersistentHover=false, ie.ensureCleanSession=false, browserName=internet explorer, enableElementCacheCleanup=true, unexpectedAlertBehaviour=dismiss, version=11, ie.usePerProcessProxy=false, ignoreProtectedModeSettings=true, cssSelectorsEnabled=true, requireWindowFocus=false, initialBrowserUrl=http://localhost:17127/, handlesAlerts=true, ie.forceCreateProcessApi=false, nativeEvents=true, browserAttachTimeout=0, ie.browserCommandLineSwitches=, takesScreenshot=true}]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:204)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:156)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:599)
	at org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:304)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.openqa.selenium.support.events.EventFiringWebDriver$2.invoke(EventFiringWebDriver.java:101)
	at $Proxy13.get(Unknown Source)
	at org.openqa.selenium.support.events.EventFiringWebDriver.get(EventFiringWebDriver.java:159)
	at com.paeobjects.home.Commonpage.getNewURL(Commonpage.java:119)
	at com.paeobjects.home.Commonpage.getFooterPage(Commonpage.java:157)
	at com.testsuite.FooterTests.FooterTests.BaseState(FooterTests.java:512)
	at com.testsuite.FooterTests.FooterTests.Footertoprankers(FooterTests.java:339)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- Footertoprankers -->
  <testcase name="Footerterms" time="51.765" classname="com.testsuite.FooterTests.FooterTests"/>
  <testcase name="Footerpress" time="75.381" classname="com.testsuite.FooterTests.FooterTests"/>
  <testcase name="Footeraboutranker" time="30.581" classname="com.testsuite.FooterTests.FooterTests"/>
  <testcase name="MobileSiteLinkCheck" time="48.520" classname="com.testsuite.FooterTests.FooterTests">
    <error type="java.lang.AssertionError" message="expected [http://m.ranker.com/] but found [http://www.ranker.com/]">
      <![CDATA[java.lang.AssertionError: expected [http://m.ranker.com/] but found [http://www.ranker.com/]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:176)
	at org.testng.Assert.assertEquals(Assert.java:186)
	at com.testsuite.FooterTests.FooterTests.MobileSiteLinkCheck(FooterTests.java:500)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- MobileSiteLinkCheck -->
  <testcase name="Footerwidget" time="30.873" classname="com.testsuite.FooterTests.FooterTests"/>
  <testcase name="FooterPartnerProgram" time="35.664" classname="com.testsuite.FooterTests.FooterTests"/>
  <testcase name="Footerfaqhelp" time="28.485" classname="com.testsuite.FooterTests.FooterTests"/>
  <testcase name="Footerjobs" time="31.204" classname="com.testsuite.FooterTests.FooterTests"/>
  <testcase name="Footersitemap" time="27.970" classname="com.testsuite.FooterTests.FooterTests"/>
  <testcase name="WikipediaLinkCheck" time="24.161" classname="com.testsuite.FooterTests.FooterTests"/>
  <testcase name="FreeBaseIconCheck" time="18.769" classname="com.testsuite.FooterTests.FooterTests"/>
  <testcase name="Footeredgecast" time="29.246" classname="com.testsuite.FooterTests.FooterTests"/>
  <testcase name="FactualIconCheck" time="38.185" classname="com.testsuite.FooterTests.FooterTests"/>
  <testcase name="Footercontact" time="29.071" classname="com.testsuite.FooterTests.FooterTests"/>
</testsuite> <!-- com.testsuite.FooterTests.FooterTests -->
